app/utils/__init__.py

```diff
@@ -127,3 +129,8 @@ def is_attachment_only(
         message.poll,
         message.stickers,
     ))
+
+
+async def suppress_embeds_after_timeout(message: dc.Message, delay: float = 5) -> None:
```

Comment:
`float` then `5` looks odd to me; it also doesn't match `remove_view_after_timeout()`.

```suggestion
async def suppress_embeds_after_timeout(message: dc.Message, delay: float = 5.0) -> None:
```